src/components/badge/bl-badge.test.ts

```diff
@@ -0,0 +1,79 @@
+import { assert, elementUpdated, expect, fixture, html } from '@open-wc/testing';
+import BlBadge from './bl-badge';
+import type typeOfBlBadge from './bl-badge';
+
+describe('bl-badge', () => {
+  const oldFetch = window.fetch;
+
+  before(() => {
+    window.fetch = async (url: RequestInfo) => {
+      if (/info\.svg$/.test(url.toString())) {
+        return new Response('<svg></svg>');
+      }
+      return new Response('', { status: 404 });
+    };
+  });
+
+  after(() => {
+    window.fetch = oldFetch;
+  });
+
+  it('is defined', () => {
+    const el = document.createElement('bl-badge');
+    assert.instanceOf(el, BlBadge);
+  });
+
+  it('renders with default values', async () => {
+    const el = await fixture<typeOfBlBadge>(html`<bl-badge></bl-badge>`);
+
+    assert.shadowDom.equal(
+      el,
+      `
+        <span class="badge">
+          <slot name="badge-icon"></slot>
+          <slot></slot>
+        </span>
+      `
+    );
+  });
+
+  it('check default values', async () => {
```

Comment:
I couldn't find a standard document about this but I always tend to write case definitions starts with `should...`. So this could be `it('should have correct default values', ...` Maybe we can agree on a standard and document on our contribution guideline.

Reply:
The button tests were written like this so I wrote the same way. I agree with your opinion, we can write like this "should ... when ..".

Reply:
I'm not sure if we need given-when-then comments but I liked the description 😊